pegasus/sites.v3/code.org/views/tenth_anniversary_tweets.html

```diff
@@ -0,0 +1,13 @@
+<div class="tenth-anniversary-tweets">
+  <blockquote class="twitter-tweet" data-dnt="true"><p lang="en" dir="ltr">Ten years ago TODAY, <a href="https://twitter.com/codeorg?ref_src=twsrc%5Etfw">@codeorg</a> was born! It’s been an awesome decade of inspiring future tech leaders and working to close the computer science opportunity gap! As education transforms in the age of AI, we hope to continue empowering students and teachers everywhere. 💻🎉 <a href="https://t.co/5IcreGlGXM">pic.twitter.com/5IcreGlGXM</a></p>&mdash; Code.org (@codeorg) <a href="https://twitter.com/codeorg/status/1629843842554789888?ref_src=twsrc%5Etfw">February 26, 2023</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```

Comment:
Ditto here –– do these need to be translated?

Reply:
Same answer as here: https://github.com/code-dot-org/code-dot-org/pull/52100#discussion_r1223562584